docs/references.bib

```diff
@@ -3657,6 +3657,19 @@ @Book{            serre1965
   url           = {https://doi.org/10.1007/978-1-4757-3910-7}
 }
 
+@Book{            serre1968,
```

Comment:
Do you think that for the benefit of our non-French-reading colleagues we should be referring to the English translation? I am ambivalent about this, I just thought I'd raise the question.

Reply:
I ended up leaving the French Corps Locaux, and adding Serre's article in Cassels-Frohlich (in English), which will at any rate be needed because of its neatest presentation of local CFT. How about that?